src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import '../styles/card';
+@import '../styles/stars';
+
+/* See no reason to add this to separate file (i probably should it) */
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property should ideally be applied to all elements using a universal selector (`*`) to ensure consistent box-sizing across the entire document. Consider adding `* { box-sizing: border-box; }` at the top of your stylesheet.